azerothcore/azerothcore-wotlk
# Fix(DB/creatures): UBRS stealthed mobs issues

## Changes Proposed:
-  removes sneak aura that was applied to the db

## Issues Addressed:

- closes https://github.com/azerothcore/azerothcore-wotlk/issues/8352

## SOURCE:


## Tests Performed:

- built and went there, mob not stealthed anymore
- 


## How to Test the Changes:


1. .go c 137990, see the mob is not in sneak anymore. Note that it's a pat, so he can be kind of wherever between that point and the bridge to Drakky (walks along the split to the beast)
2. Aggro any of the packs that have assassins (the ones after the Rend event), and see the assassins only remove stealth on attacking.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
